README.md

```diff
@@ -57,6 +57,9 @@ reused in the target application to speed up the design of the media IO layer (d
 
 ## Example
 
+## TODO: Update to show how to use the new sized de-allocations, and how not to confuse payload_size and allocated_size.
+☝ todo will be addressed as soon as the new API is stable (tx, rx, and "zero copy" aspects).
```

Comment:
Maybe it even makes sense to introduce a dedicated new section about migration from v3 -> v4 ...

Reply:
Yes indeed. I think it can be entirely written by o1 if you feed it the diff.